### PR TITLE
Destination bigquery: add raw override test for gcs mode

### PR DESCRIPTION
**Pull request by edgao**
created a new CI secret here https://console.cloud.google.com/security/secret-manager/secret/SECRET_DESTINATION-BIGQUERY_CREDENTIALS_1S1T_GCS_RAW_OVERRIDE/versions?project=dataline-integration-testing

### CODE REVIEW

**Review by jbfbell — APPROVED** [review]

File: airbyte-integrations/connectors/source-datadog/integration_tests/abnormal_state.json [hunk 1]
```diff
@@ -6,4 +6,4 @@
       "stream_descriptor": { "name": "logs" }
     }
   }
-]
\ No newline at end of file
+]
```
Comment (edgao) [inline airbyte-integrations/connectors/source-datadog/integration_tests/abnormal_state.json:9]: autoformat is committing changes to random connectors... :shrug:

Comment (edgao) [reply airbyte-integrations/connectors/source-datadog/integration_tests/abnormal_state.json:9]: I feel like that's not supposed to happen?

Comment (edgao) [reply airbyte-integrations/connectors/source-datadog/integration_tests/abnormal_state.json:9]: oh, probably from https://github.com/airbytehq/airbyte/pull/29798/files

**Comment by octavia-approvington** [comment]
Approve this <br/> imagine the commander saying yes

**Review by octavia-approvington — APPROVED** [review]

**Pull request merged**